Client/src/components/Pages/Explore/PreviousCities.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import getCitiesFromLS from '../../../services/getCitiesFromLS';
+import SearchResult from '../../layout/Search/SearchResult';
+import styles from './PreviousCities.modules.css';
+function PreviousCities() {
+  const [lastCities, setLastCities] = useState();
+
+  useEffect(async () => {
+    setLastCities(await getCitiesFromLS());
```

Comment:
It would be better in terms of readability to:
```
const recentCities = await getRecentCities();
setRecentCities(recentCities);
```

Mayby `recent` is a bit more suitable word here...